components/DetailsPageHighlights/DetailsPageHighlights.js

```diff
@@ -0,0 +1,58 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import MarkdownRenderer from '../MarkdownRenderer/MarkdownRenderer'
```

Comment:
From our other React library, we've been using an `internal` folder for components that aren't really for general consumption. The MarkdownRenderer might belong there instead